From 01ed93aa5bd9730a2788fc50ffa83dc0a97e5e6c Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@freescale.com>
Date: Sun, 14 Apr 2013 20:25:46 +0530
Subject: [PATCH 270/767] usb: gadget: Turn on cache snooping hardware for
 64-bit Kernel

Turn on cache snooping hardware for 64-bit Kernel

Signed-off-by: Suresh Gupta <suresh.gupta@freescale.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/usb/gadget/udc/fsl_udc_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/udc/fsl_udc_core.c b/drivers/usb/gadget/udc/fsl_udc_core.c
index be59309e848c..426ce8958da1 100644
--- a/drivers/usb/gadget/udc/fsl_udc_core.c
+++ b/drivers/usb/gadget/udc/fsl_udc_core.c
@@ -332,7 +332,7 @@ static int dr_controller_setup(struct fsl_udc *udc)
 	}
 #endif
 
-#if defined(CONFIG_PPC32) && !defined(CONFIG_NOT_COHERENT_CACHE)
+#if !defined(CONFIG_NOT_COHERENT_CACHE)
 	/* Turn on cache snooping hardware, since some PowerPC platforms
 	 * wholly rely on hardware to deal with cache coherent. */
 
-- 
2.17.0

